recipe/meta.yaml

```diff
@@ -61,21 +59,20 @@ requirements:
         - {{ cdt('libxau') }}                # [linux]
         - {{ cdt('libxxf86vm') }}            # [linux]
     host:
-        - libblas
-        - libcblas
-        - liblapack
-        - libglu                  # [linux]
-        - freeglut                # [linux]
-        - xorg-libxmu             # [linux]
+        - openblas  # [win]
+        - libblas  # [not win]
+        - liblapack  # [not win]
```

Comment:
I'm not sure about this myself, but should we build against openblas on all OSes? Otherwise, is libblas and liblapack the less optimized versions?

Reply:
I tried using openblas on all OSes, but ran into issues on UNIX that I didn't know how to resolve. I am not sure how optimized conda's libblas & liblapack libraries are (there seem to be multiple versions), but I once tested Simbody with optimized vs non-optimized blas/lapack a long time ago and it didn't make much difference.

I created #32 to track this.